pkg/azclient/cloud_test.go

```diff
@@ -236,5 +236,32 @@ var _ = ginkgo.Describe("Cloud", func() {
 				os.Unsetenv(azclient.EnvironmentFilepathName)
 			})
 		})
+		ginkgo.When("the environment is set,file is correct", func() {
+			ginkgo.It("should return error", func() {
+				configFile, err := os.CreateTemp("", "azure.json")
+				gomega.Expect(err).ToNot(gomega.HaveOccurred())
+				defer os.Remove(configFile.Name())
+
+				err = os.WriteFile(configFile.Name(), []byte(`
+				{
+                   "resourceManagerEndpoint":"https://management.chinacloudapi.cn",
```

Comment:
could you format the indention?

Reply:
<img width="428" alt="image" src="https://github.com/user-attachments/assets/2b952388-e668-4d10-9069-78fcc0eb28d8" />

Reply:
But I got this when I open it with vscode